app/jobs/itms/fetch_all_data.rb

```diff
@@ -0,0 +1,69 @@
+class Itms::FetchAllData < ApplicationJob
+  queue_as :itms
+
+  def self.perform_all_harvesters
+    harvesters = [
+      {
+        harvester: Itms::AccountingDocumentsHarvester,
+        object: Itms::AccountingDocument
+      },
+      {
+        harvester: Itms::ApprovedZonfpHarvester,
+        object: Itms::ApprovedZonfp
+      },
+      {
+        harvester: Itms::DeclaredAppealsHarvester,
+        object: Itms::DeclaredAppeal
+      },
+      {
+        harvester: Itms::FinishedProjectsHarvester,
+        object: Itms::FinishedProject
+      },
+      {
+        harvester: Itms::OngoingProjectsHarvester,
+        object: Itms::OngoingProject
+      },
+      {
+        harvester: Itms::OperationalProgramsHarvester,
+        object: Itms::OperationalProgram
+      },
+      {
+        harvester: Itms::PaidZopHarvester,
+        object: Itms::PaidZop
+      },
+      {
+        harvester: Itms::PlannedAppealsHarvester,
+        object: Itms::PlannedAppeal
+      },
+      {
+        harvester: Itms::ProcurementsHarvester,
+        object: Itms::Procurement
+      },
+      {
+        harvester: Itms::ReceivedZonfpHarvester,
+        object: Itms::ReceivedZonfp
+      },
+      {
+        harvester: Itms::RejectedZonfpHarvester,
+        object: Itms::RejectedZonfp
+      },
+      {
+        harvester: Itms::RejectedZopHarvester,
+        object: Itms::RejectedZop
+      },
+      {
+        harvester: Itms::SubmittedZopHarvester,
+        object: Itms::SubmittedZop
+      },
+    ]
+
+    harvesters.each do |harvester|
+      Itms::FetchAllData.perform_later(harvester[:harvester].to_s, harvester[:object].to_s)
```

Comment:
Toto uplne nerozumiem preco si rozdelil na `harvesters` folder, ja by som to nechal v `jobs` ved su to joby ako vsetky ine.